presbuzz-frontend/src/components/Articles.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+
+class Articles extends Component {
+  render() {
+    return (
+      <div>
+        <h1>Article List {this.props.ids}</h1>

```

Comment:
Hard coded.
Use a generator/Loop.
Create a functional component called Article and render it here